client/my-sites/checkout/composite-checkout/components/checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -94,8 +95,10 @@ export function CheckoutSidebarPlanUpsell() {
 		{ strong: createElement( 'strong' ) }
 	);
 
+	const isEnglish = ( config( 'english_locales' ) as string[] ).includes( locale || '' );
```

Comment:
I know this is done like this elsewhere, but just for the sake of defensive coding, I wonder if it would be better to cover the case where `english_locales` is not defined for some reason:

```suggestion
	const englishLocales = config( 'english_locales' );
	const isEnglish = ( Array.isArray( englishLocales ) ? englishLocales : [] ).includes(
		locale || ''
	);
```

But I leave the decision to you.